pkg/nodeconfig/nodeconfig.go

```diff
@@ -193,13 +194,14 @@ func (nc *nodeConfig) Configure() error {
 			return errors.Wrap(err, "configuring WICD failed")
 		}
 		// Set the desired version annotation, communicating to WICD which Windows services configmap to use
-		if err := nc.applyLabelsAndAnnotations(nil, map[string]string{DesiredVersionAnnotation: version.Get()}); err != nil {
+		if err := nc.applyLabelsAndAnnotations(nil, map[string]string{DesiredVersionAnnotation: wmcoVersion}); err != nil {
```

Comment:
Do we use `applyLabelsAndAnnotations()` only for the version now? If yes, then we should move that to metadata package and make it specific to the version.

Reply:
Yep, we also use it for [other labels/annotations WMCO applies](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/nodeconfig/nodeconfig.go#L181). But let me see if I can reuse what we have in the metadata pkg to get rid of this method